lib/mayaUsd/ufe/UsdAttribute.cpp

```diff
@@ -371,6 +371,15 @@ std::string UsdAttribute::name() const
     return _attrHolder->name();
 }
 
+#ifdef UFE_V4_FEATURES_AVAILABLE
```

Comment:
Shouldn't this be 

`#ifdef UFE_HAS_DISPLAY_NAME`

instead?

Reply:
The function exists based on V4/not V4 but has different name (leading underscore or not). It also needs to align with what is done in the header, which cannot have ifdef in the middle of a define.

Reply:
Basically, in one case there is a function with leading _ which contain the real call,, in the other, the one without leading _ contains the real call. It is all confusing.